homeassistant/components/icloud/account.py

```diff
@@ -105,12 +106,18 @@ def setup(self) -> None:
             return
 
         user_info = None
+        api_devices = {}
```

Comment:
Isn't this default weird since it's a different type?

Reply:
it is